Enterprise-CMCS/macpro-mdct-qmr
# mdct 2249 - change deviations section to only include radio buttons and a textarea

### Description
This is a change only to 2023 -- when asked if there are deviations from the measure specifications, the user is prompted to answer Yes or No. If 'yes', a textArea appears and they need to describe the deviation. If they do not fill out the textArea, an error will appear at the bottom after the user clicks the `validate` button.


### Related ticket(s)


MDCT-2249

---
### How to test
Go into a measure (AAB-AD will work). Fill out the page and after you get to 'Performance Measures' you'll see the ability to add rates. Fill out one of the rates and then after you'll see a section called 'Deviations from Measure Specifications' click 'yes', then you should only see at textArea. See that leaving it blank will result in a validation error after clicking 'validate', and see that no error occurs when you fill out the textArea. 


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
